crypto/stateproof/verifier_test.go

```diff
@@ -155,3 +155,27 @@ func TestEqualVerifiers(t *testing.T) {
 
 	a.Equal(verifierLnP, verifier)
 }
+
+func TestTreeDepth(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	a := require.New(t)
+
+	p := generateProofForTesting(a, false)
```

Comment:
nits: will it be too hard/takes too much time to actually create a stateproof with >20 tree depth?

Reply:
I can't make it run on my local machine. Do we need such an excessive test?